src/python/grpcio/grpc/aio/_interceptor.py

```diff
@@ -543,7 +551,7 @@ async def _proxy_writes_as_request_iterator(self):
             yield value
 
     async def _write_to_iterator_queue_interruptible(
-        self, request: RequestType, call: InterceptedCall
+        self, request: object, call: InterceptedCall
```

Comment:
We should avoid using `object` as type annotation as it doesn't provide much value.